pkgs/development/python-modules/simplesat/default.nix

```
@@ -0,0 +1,51 @@
+{ buildPythonPackage
+, fetchFromGitHub
+, writeText
+, lib
+, attrs
+, six
+, okonomiyaki
+}:
+
+let
+
+  # To regenerate:
+  # git clone https://github.com/enthought/sat-solver
+  # cd sat-solver
+  # nix-shell -p python3 python3Packages.setuptools --run "python3 ./setup.py build"
+  # cat simplesat/_version.py
+  versionFile = writeText "simplesat_ver" ''
+    # THIS FILE IS GENERATED FROM SETUP_EXT
+    version = '0.8.2'
+    full_version = '0.8.2'
+    git_revision = '4ce881644c872e629358e31f37bc8d099ce42591'
+    is_released = True
+
+    msi_version = '0.8.2.843'
+
+    version_info = (0, 8, 2, 'final', 0)
+  '';
+
+in buildPythonPackage rec {
+  pname = "simplesat";
+  version = "0.8.2";
+
+  propagatedBuildInputs = [ attrs six okonomiyaki ];
+
+  src = fetchFromGitHub {
+    owner = "enthought";
+    repo = "sat-solver";
+    rev = "v${version}";
+    hash = "sha256-6BQn1W2JGrMmNqgxi+sXx06XzNMcvwqYGMkpD0SSpT8=";
+  };
+
+  preConfigure = "cp ${versionFile} simplesat/_version.py";
+  setuptoolsCheckPhase = "true";
```

```suggestion
  dontUseSetuptoolsCheck = true;
```
why do we set this? Do you want to disable tests? If so, use doCheck.